app/assets/stylesheets/mtl/extend/_forms.scss

```diff
@@ -17,3 +17,37 @@
 .input-field.col label {
   padding-right: $gutter-width;
 }
+
+// fields with text suffix/unit
+.input-field.has-suffix {
+  input:not([type]),
```

Comment:
because this is the very same way the mtl library implements it. their way excludes anything inexplicit. If I use input only, I'd have to normalize radios and checkboxes afterwards.